lib/index.d.ts

```diff
@@ -361,7 +361,14 @@ declare namespace Realm.Sync {
         expiresAt?: Date;
     }
 
-    type ErrorCallback = (message?: string, isFatal?: boolean, category?: string, code?: number) => void;
+    interface SyncError {
+        message: string; 
```

Comment:
Is only `category` optional?

Reply:
i think so. from the code it seems all other will be present with some value